e2eTests/e2eTestHelper.psm1

```diff
@@ -304,9 +402,13 @@ function CommitAndPush {
 
 function MergePRandPull {
     Param(
+        [string] $repository,
         [string] $branch = "main"
     )
 
+    if (!$repository) {
+        $repository = $defaultRepository

```

Comment:
Can't `$defaultRepository` be set as default value for `$repository` param?

Reply:
Default parameters are only used if the parameter isn't specified.
In this case, the repository might be transferred as empty.